utils/mtpoe-ctrl/Makefile

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (C) 2018 adron@yapic.net
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mtpoe-ctrl
+PKG_SOURCE_VERSION:=9c42b74274cca10810363dbbb6d161bfb9cedb7e
+PKG_MIRROR_HASH:=1ac2902161ba4bb1e8af397aa48821ccea95740199b1c155f98161c7ff3a93df
```

Comment:
This ordering is not correct, you should check how it is done in other Makefiles.

Reply:
It also comes from the original repo approach. Will see.